scripts/README.rst

```diff
@@ -0,0 +1,78 @@
+Feature Toggle Reporter:
+------------------------
+
+We make extensive use of a variety of feature toggles at edX. However, it has
+the downside of making the state of a deployed application somewhat opaque.
+This tool anaylzes data from the codebases of deployed applications, as well
+as their databases, to provide a summary of the feature toggles in use in a
+given deployment. For more information, please see:
+
+https://open-edx-proposals.readthedocs.io/en/latest/oep-0017-bp-feature-toggles.html
+
+Prerequisites:
+--------------
+
+In order to create a feature toggle report for a given deployment, you need
+two types of data: feature toggle annotation data and feature toggle data from
+an application's database
+
+Feature Toggle Annotation Data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Feature toggles are first defined in the code of an application. We make use of
+the `code_annotations`_ tool, to annotate the definitions of these components
+with information regarding their purpose, default values and planned expiration
+dates. This data is then gathered into report. The feature toggle reporter uses
+this data.
+
+.. code:: bash
+
+    code_annotations static_find_annotations --config_file feature_annotations.yaml
+
+Rename the resulting yml file `<ida_name>_annotation_data.yml`. This step is
+necessary, as the feature toggler report generator will key off the `ida_name`
+in the filename in order to be able to link this data to the SQL data collected
```

Comment:
If it's been captured as a YAML file, "SQL data" doesn't really seem like an appropriate term.  Maybe "toggle state data"?